aml_module/invoker.py

```diff
@@ -38,10 +40,20 @@ def is_timestamp_ascending(timestamps):
 
 def invoke(input_path, detect_mode, timestamp_column, value_column, batch_size, threshold, sensitivity,
             appendMode, compute_stats_in_visualization, output_path):
-    data_frame_directory = load_data_frame_from_directory(input_path)
+    surfix = input_path.split('/')[-1].split('.')[-1]
 
+    if surfix != 'csv':
+        logger.debug('CSV file not found')
+        raise UserError(InvalidInputType)
+
+    df = pd.read_csv(input_path)
+    save_data_frame_to_directory(output_path, df)
+    data_frame_directory = load_data_frame_from_directory(output_path)
```

Comment:
data_frame_directory = DataFrameDirectory.create(data=df)
no need to save then load.

Reply:
Thanks! will fix it.